channels/us_pluto.m3u

```diff
@@ -1325,8 +1327,8 @@ https://service-stitcher.clusters.pluto.tv/v1/stitch/embed/hls/channel/5f9847fd5
 http://pluto-live.plutotv.net/egress/chandler/pluto01/live/VIACBS07/master.m3u8
 #EXTINF:-1 tvg-id="PlutoTVMTVSpankinNew.us" tvg-country="US" tvg-language="English" tvg-logo="http://images.pluto.tv/channels/5d14fdb8ca91eedee1633117/colorLogoPNG.png" group-title="Series",Pluto TV MTV Spankin' New (720p)
 https://service-stitcher.clusters.pluto.tv/stitch/hls/channel/5d14fdb8ca91eedee1633117/master.m3u8?advertisingId=&appName=web&appStoreUrl=&appVersion=DNT&app_name=&architecture=&buildVersion=&deviceDNT=0&deviceId=5d14fdb8ca91eedee1633117&deviceLat=&deviceLon=&deviceMake=web&deviceModel=web&deviceType=web&deviceVersion=DNT&includeExtendedEvents=false&marketingRegion=US&serverSideAds=false&sid=869&terminate=false&userId=
-#EXTINF:-1 tvg-id="PlutoTVMTVTeenMom.us" tvg-country="US" tvg-language="English" tvg-logo="https://images.pluto.tv/channels/5cffcf5686dfe15595fb3f56/featuredImage.jpg?w=750&fm=png" group-title="Series",Pluto TV MTV Teen Mom (720p)
```

Comment:
Channel link still valid to this date, can serve as alternative to the one below.

Reply:
OK.

Reply:
@LaneSh4d0w Please, merge it now.

Reply:
Fixed.